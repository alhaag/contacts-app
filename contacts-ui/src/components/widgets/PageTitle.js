// third party
import React from 'react'


const PageTitle = (props) => {
    const { title } = props;
    return <h2 className="md-display-1">{props.title}</h2>
}

export default PageTitle
