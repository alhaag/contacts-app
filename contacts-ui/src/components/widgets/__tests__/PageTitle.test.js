import React from 'react'
import renderer from 'react-test-renderer'
import PageTitle from '../PageTitle';

test('Presentation PageTitle component', () => {

  const component = renderer.create(
    <PageTitle title="Test Title" />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  // manually trigger the callback
  /*tree.props.onMouseEnter();
  // re-rendering
  tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  // manually trigger the callback
  tree.props.onMouseLeave();
  // re-rendering
  tree = component.toJSON();
  expect(tree).toMatchSnapshot();*/
})