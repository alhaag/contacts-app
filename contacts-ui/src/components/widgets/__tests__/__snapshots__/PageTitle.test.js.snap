// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Presentation PageTitle component 1`] = `
<h2
  className="md-display-1"
>
  Test Title
</h2>
`;
